### PR TITLE
feat: add PR title check workflow

### PR DESCRIPTION
**Pull request by georgi-l95**
**Description**:
This PR adds workflow, which is responsible for checking the title of the PR and if it's compliant with the Conventional Commits standard

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by isavov — APPROVED** [review]

**Review by AlfredoG87 — CHANGES_REQUESTED** [review]
Could we add a note on `README.md` with what is that standard?

Maybe under the `Contributing` section?

**Comment by georgi-l95** [comment]
@AlfredoG87 Agree with you, but I think this should be included in the contribution guide, more so because all PRs in all repos will need to comply with this rule. Not sure, if we should clutter the readme here.

**Review by AlfredoG87 — APPROVED** [review]

**Comment by Nana-EC** [comment]
> > @AlfredoG87 Agree with you, but I think this should be included in the contribution guide, more so because all PRs in all repos will need to comply with this rule. Not sure, if we should clutter the readme here.
> 
> yeah, I agree with you that this should be on contributing guide. do we have one? 😅

Yeah you can open a ticket here https://github.com/hashgraph/.github/blob/main/CONTRIBUTING.md

File: .github/workflows/flow-pr-title-check.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+jobs:
+  title-check:
+    name: Title Check
+    runs-on: ubuntu-latest
+    permissions:
+      statuses: write
+    steps:
+      - name: Check PR Title
+        uses: aslafy-z/conventional-pr-title-action@v3
```
Comment (Nana-EC) [inline .github/workflows/flow-pr-title-check.yml:43]: Q: does this support things like checking if labels are assigned and milestones set etc?

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**